applications/asset_tracker_v2/src/cloud/cloud_codec/cloud_codec.h

```diff
@@ -255,29 +259,94 @@ enum cloud_codec_event_type {
 };
 
 struct cloud_codec_evt {
+	/** Cloud codec event type */
 	enum cloud_codec_event_type type;
+	/** New config data if type is CLOUD_CODEC_EVT_CONFIG_UPDATE */
```

Comment:
I would rather update the documentation for the `CLOUD_CODEC_EVT_CONFIG_UPDATE` event to include the payload type parameters. Also include that `CLOUD_CODEC_EVT_CONFIG_UPDATE` is only used for the LwM2M build. This aligns well with how its done in other header files, for instance in `lte_lc.h` or `modem_module_event.h`.